desc/plotting.py

```diff
@@ -1282,6 +1282,132 @@ def plot_surfaces(eq, rho=8, theta=8, zeta=None, ax=None, **kwargs):
     return fig, ax
 
 
+def plot_boundary(eq, zeta=None, axis=True, ax=None, **kwargs):
+    """Plot stellarator boundary at multiple toroidal coordinates.
+
+    Parameters
+    ----------
+    eq : Equilibrium
+        Object from which to plot.
+    zeta : int or array-like or None
+        Values of zeta to plot boundary surface at.
+        If an integer, plot that many contours linearly spaced in [0,2pi).
+        Default is 1 contour for axisymmetric equilibria or 4 for non-axisymmetry.
+    axis : bool
+        Whether or not to plot the magnetic axis locations. Default is True.
+    ax : matplotlib AxesSubplot, optional
+        Axis to plot on.
+    **kwargs : fig,ax and plotting properties
+        Specify properties of the figure, axis, and plot appearance e.g.::
+
+            plot_X(figsize=(4,6),label="your_label")
+
+        Valid keyword arguments are:
+
+        figsize: tuple of length 2, the size of the figure (to be passed to matplotlib)
+        cmap : colormap to use for plotting, discretized into len(zeta) colors
+        colors: array of colors to use for each zeta angle
+        ls : array of line styles to use for each zeta angle
+        lw : array of line widths to use for each zeta angle
+        marker: str, marker style to use for the axis plotted points
+        size: float, marker size to use for the axis plotted points
+        label_fontsize: float, fontsize of the x and y labels
+        legend_fontsize: float, fontsize of the legend
+
+    Returns
+    -------
+    fig : matplotlib.figure.Figure
+        Figure being plotted to.
+    ax : matplotlib.axes.Axes or ndarray of Axes
+        Axes being plotted to.
+
+    Examples
+    --------
+
+    .. image:: ../../_static/images/plotting/plot_boundary.png
+
+    .. code-block:: python
+
+        from desc.plotting import plot_boundary
+        fig, ax = plot_boundary(eq)
+
+    """
+    figsize = kwargs.pop("figsize", None)
+    cmap = kwargs.pop("cmap", "rainbow")
+    colors = kwargs.pop("colors", None)
+    ls = kwargs.pop("ls", None)
+    lw = kwargs.pop("lw", None)
+    marker = kwargs.pop("marker", "x")
+    size = kwargs.pop("size", 36)
+    label_fontsize = kwargs.pop("label_fontsize", None)
+    legend_fontsize = kwargs.pop("legend_fontsize", None)
+
+    assert (
+        len(kwargs) == 0
+    ), f"plot surfaces got unexpected keyword argument: {kwargs.keys()}"
+
+    if zeta is None:
+        if eq.N == 0:
+            zeta = 1
+        else:
+            zeta = 4
```

Comment:
another minor thing: for most of the other stellarator plots we default to 6 cross sections, should that be the same here, or do we want to change it elsewhere?

Reply:
We default to 6 cross-sections when they are in separate subplots. I think 4 is a good default for this function that plots them all in the same subplot, because otherwise it starts to get too crowded.